docs/pages/guides/monorepos.mdx

```diff
@@ -217,15 +217,15 @@ Like standard packages, we need to add our **cool-package** as a dependency to o
   },
   "dependencies": {
     "cool-package": "*",
-    "expo": "~50.0.0",
-    "expo-status-bar": "~1.10.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-native": "0.73.0",
-    "react-native-web": "~0.19.13"
+    "expo": "~53.0.0",
+    "expo-status-bar": "~2.2.3",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "react-native": "0.79.4",
+    "react-native-web": "~0.20.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0"
+    "@babel/core": "^7.52.2"
```

Comment:
This seems to be incorrect package version. The latest version published in npm registry is 7.27.7:
* https://www.npmjs.com/package/@babel/core